services/motion/builtin/move_on_globe_test.go

```diff
@@ -314,7 +314,7 @@ func TestObstacleReplanningGlobe(t *testing.T) {
 				// Note: for CreateMoveOnGlobeTestEnvironment, the camera is given an orientation such that it is pointing left, not
 				// forwards. Thus, an obstacle in front of the base will be seen as being in +X.
 				obstaclePosition := spatialmath.NewPoseFromPoint(r3.Vector{X: 300, Y: 0, Z: 0})
-				box, err := spatialmath.NewBox(obstaclePosition, r3.Vector{X: 20, Y: 20, Z: 10}, caseName)
+				box, err := spatialmath.NewBox(obstaclePosition, r3.Vector{X: 40, Y: 40, Z: 10}, caseName)
```

Comment:
I changed these values because this test became flaky.
I also changed the planopts.resolution value from 60 to 30 mm within `check.go` so that we have finer interpolations.

e.g. our transformed obstacle would be at `x:330, y: -8` and our interpolation would go from 
`x:290` to `x:350` effectively missing the obstacle collision when it would've been caught with more in-between steps.

Reply:
Makes sense, nice catch